hutool-core/src/test/java/cn/hutool/core/collection/IterUtilTest.java

```diff
@@ -2,16 +2,22 @@
 
 import java.util.ArrayList;
 import java.util.Map;
+import java.util.LinkedHashMap;
+import java.util.HashMap;
 
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.Rule;
 
 /**
  * {@link IterUtil} 单元测试
  * @author looly
  *
  */
 public class IterUtilTest {

```

Comment:
I would say that these tests are less interesting than the rest but the choice is yours.